llvm/lib/Analysis/VectorUtils.cpp

```diff
@@ -1477,6 +1480,37 @@ void VFABI::getVectorVariantNames(
   }
 }
 
+std::optional<FunctionType *>
```

Comment:
Can this just be `FunctionType *`? We don't need to distinguish between `std::nullopt` and `nullptr` so I don't think the optional provides any value.